universal-login-sdk/lib/config.js

```diff
@@ -1,6 +1,14 @@
+import {OPERATION_CALL} from 'universal-login-contracts';
+
 const DEFAULT_PAYMENT_OPTIONS = {
   gasPrice: 1000000000,
   gasLimit: 1000000
 };
 
-export default DEFAULT_PAYMENT_OPTIONS;
+const DEFAULTS = {
+  operationType: OPERATION_CALL,
+  value: 0,
+  data: '0x0'
+};
```

Comment:
Why not have just one`DEFAULTS`, why split into `DEFAULTS` and `DEFAULT_PAYMENT_OPTIONS`.

btw naming: `MESSAGE_DEFAULTS`

Reply:
✅